### PR TITLE
Docx implementation

### PR DESCRIPTION
**Pull request by Rafalkufel**
… Introduction webpack server to render electron content.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({value: event.target.value});
+  }
+  
+  handleSubmit(event) {
+    console.log(this.state.value);
```
Comment (Maciek246) [inline src/App.js]: Please, remove console.log()

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,38 @@
...
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        <form onSubmit={this.handleSubmit}>
+          <label>
+            File name:
+            <input type="text" value={this.state.value} onChange={this.handleChange} />
```
Comment (Maciek246) [inline src/App.js]: Is input should be part of a label?

```suggestion
<form onSubmit={this.handleSubmit}>
  <label htmlFor="fileName">File name</label>
  <input type="text" name="fileName" value={this.state.value} onChange={this.handleChange} />
  <input type="submit" value="Submit" />
</form>
```

File: src/decision.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const docx = require('docx');
+const fs = require('file-system');
```
Comment (Maciek246) [inline src/decision.js]: You can use an `import` statement instead `require`

File: src/decision.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  save() {
+    var packer = new Packer();
+    packer.toBuffer(this.doc).then((buffer) => {
+      fs.writeFile(`${this.value}.docx`, buffer)
+    });
+  }
+}
+
+module.exports = DecisionCreate;
```
Comment (Maciek246) [inline src/decision.js]: New line

**Review by Maciek246 — CHANGES_REQUESTED** [review]

**Review by Maciek246 — APPROVED** [review]

File: src/decision.js (hunk 3 above)
Comment (Rafalkufel) [reply src/decision.js]: Unfortunately node doesn't support `import` statement.

**Pull request merged**